File: Java/calculator.java
import java.util.Scanner;

public class calculator {
    public static void main(String[] args) {
        try (Scanner scan = new Scanner(System.in)) {
            System.out.print("Please enter the operand(+,-,*,/) to perform operation in JAVA calculator: ");
            char input = scan.next().charAt(0);

            if (input == '+') {
                System.out.println("You have opted for summation.");
            } else if (input == '-') {
                System.out.println("You have opted for difference.");
            } else if (input == '*') {
                System.out.println("You have opted for multiplication.");
            } else if (input == '/') {
                System.out.println("You have opted for division.");
            } else {
                System.out.println("You have entered wrong input.");
            }

            System.out.print("Enter fisrt number: ");
            float num1 = scan.nextFloat();
            System.out.print("Enter second number: ");
            float num2 = scan.nextFloat();

            switch (input) {
                case '+':
                    System.out.print("The sum is: ");
                    System.out.println(num1 + num2);
                    break;
                case '-':
                    System.out.print("The difference is: ");
                    System.out.println(num1 - num2);
                    break;
                case '*':
                    System.out.print("The multiplication is: ");
                    System.out.println(num1 * num2);
                    break;
                case '/':
                    System.out.print("The division is: ");
                    System.out.println(num1 / num2);
                    break;
            }
        }
    }
}

File: C/calculator.c
#include <stdio.h>

int main()
{
    char calculator;
    float a, b;
    printf("Choose an operator(+, -, *, /) to perform the operation in C Calculator: ");
    scanf("%c", &calculator);
    if (calculator == '+')
    {
        printf("You have selected summation.\n");
    }
    else if (calculator == '-')
    {
        printf("You have selected difference.\n");
    }
    else if (calculator == '*')
    {
        printf("You have selected multiplication.\n");
    }
    else if (calculator == '/')
    {
        printf("You have selected division.\n");
    }

    printf("Enter number 1: ");
    scanf("%f", &a);
    printf("Enter number 2: ");
    scanf("%f", &b);

    switch (calculator)
    {
    case '+':
        printf("The sum is: %f", a + b);
        break;
    case '-':
        printf("The difference is: %f", a - b);
        break;
    case '*':
        printf("The multiplication is: %f", a * b);
        break;
    case '/':
        printf("The division is: %f", a / b);
        break;
    default:
        break;
    }
    return 0;
}